controllers/api/server.go

```diff
@@ -71,6 +71,8 @@ func (as *Server) registerRoutes() {
 	router.HandleFunc("/import/group", as.ImportGroup)
 	router.HandleFunc("/import/email", as.ImportEmail)
 	router.HandleFunc("/import/site", as.ImportSite)
+  router.HandleFunc("/webhooks/", mid.Use(as.Webhooks, mid.RequirePermission(models.PermissionModifySystem)))
```

Comment:
You might want to add `goimports` to your IDE settings to run automatically on save. I've found this helps me from having to go back later and reformat all the files.